subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -84,6 +84,16 @@ Some builds may have worked around this by also putting the additional attribute
 
 The embedded Kotlin has been updated to link:https://github.com/JetBrains/kotlin/releases/tag/v1.8.21[Kotlin 1.8.21].
 
+[[unsupported_ge_plugin_3.13]]
+==== Earliest supported Gradle Enterprise plugin version is 3.13.1
+
+Starting in Gradle 9.0, the earliest supported Gradle Enterprise plugin version is 3.13.1.
```

Comment:
I am not sure how we should add this to the upgrading guide. I remember that we added an entry for the deprecation, and then another entry under the 9.0 section for the actual removal of the deprecated behavior. I wonder if we should do the same here or not. That would then mean that we'd have a section "Deprecated usage of Gradle Enterprise plugin version prior to 3.13.1". WDYT?

Reply:
@wolfs: agreed, this entire paragraph should be moved to the **Deprecations** section of the doc.

Reply:
It seems, when I rebased onto recent `master`, the section somehow moved under **Deprecations** automatically.

@lkasso, PTAL at how it is now, but also, please push directly to this PR branch, if you feel any more changes are necessary to this file.